quickbooks/2.0/modules/ROOT/pages/quickbooks-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,417 @@
+= Upgrade Quickbooks Connector to Version 2.0 - Mule 4
+
+Upgrade Anypoint Connector for Quickbooks (Quickbooks Connector) to version 2.x.
+
+== Supported Upgrade Paths
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|1.x |2.x
+|===
+
+== Changes in This Release
+
+This release contains the following changes:
+
+* Made the following changes to some operations:
+** <<changed_operations, Changed operation names>>
+** The *Get query* operation now makes an HTTP GET request instead of an HTTP POST request to the /query endpoint
+** The *Get invoice by invoice id* operation now makes an HTTP GET request instead of an HTTP POST request to the /invoice/{invoiceId} endpoint
```

Comment:
```suggestion
** The *Get invoice by invoice id* operation now makes an HTTP GET request instead of an HTTP POST request to the `/invoice/{invoiceId}` endpoint
```